source/lib-intro.tex

```diff
@@ -646,14 +646,29 @@
 
 \begin{itemize}
 \item
+Properties specified as locale-specific may change during program execution
+by a call to \tcode{setlocale(int, const char*)}\iref{clocale.syn}, or
+by a change to a \tcode{locale} object,
+as described in \ref{locales} and \ref{input.output}.
+\item
+The \defnadj{execution}{character set} and
+the \defnadj{execution}{wide-character set}
```

Comment:
Problem with wording in paper: I can't tell what these terms mean based on this definition. What are they? Should we open an issue?

Reply:
Feel free to file an LWG issue, but this just moves these not-quite definitions from the core wording to the library wording, so I'd say this is pre-existing.

Reply:
The "implementation-defined" part was lost in the move. Is that intended?

Reply:
The paper shows the "implementation-defined" part as removed. I don't think I intended to remove "implementation-defined", although it serves little purpose given that the only observable thing here is the encoding (which is locale-specific).

Reply:
@tkoeppe, what do you think?

Reply:
Not particularly, but I think it helps with the "what these character sets are" question. We can sort this out in the LWG issue Dawn filed.